site/sidebars.js

```diff
@@ -11,16 +11,76 @@
 
 module.exports = {
   // By default, Docusaurus generates a sidebar from the docs folder structure
-  tutorialSidebar: [{ type: 'autogenerated', dirName: '.' }],
-
-  // But you can create a sidebar manually
-  /*
-  tutorialSidebar: [
+  docs: [
     {
       type: 'category',
-      label: 'Tutorial',
-      items: ['hello'],
+      label: 'Docs',
+      collapsible: true,
+      collapsed: true,
+      items: [
+        'getting-started',
+        {
+          type: 'category',
+          label: 'Sample Queries',
+          collapsible: true,
+          collapsed: true,
+          items: [
+            {
+              type: 'autogenerated',
+              dirName: 'sample-queries',
+            },
+          ],
+        },
+        {
+          type: 'category',
+          label: 'How-to guides',
+          collapsible: true,
+          collapsed: true,
+          items: [
+            {
+              type: 'autogenerated',
+              dirName: 'how-to',
+            },
+          ],
+        },
+        {
+          type: 'category',
+          label: 'Concepts',
+          collapsible: true,
+          collapsed: true,
+          items: [
+            {
+              type: 'autogenerated',
+              dirName: 'concepts',
+            },
+          ],
+        },
+        {
+          type: 'category',
+          label: 'ECS Fargate tutorials',
+          collapsible: true,
+          collapsed: true,
+          items: [
+            {
+              type: 'autogenerated',
+              dirName: 'fargate-tutorials',
+            },
+          ],
+        },
+        {
+          type: 'category',
+          label: 'SQL reference',
+          collapsible: true,
+          collapsed: true,
+          items: [
+            {
+              type: 'doc',
+              label: 'SQL reference',
+              id: 'sql/sql_reference',
```

Comment:
why not just `reference`?

Reply:
I saw that reference generates the link but does not add it to navigation. Didn't test however, it may have worked as well.